ONSdigital/eq-author-app
# Update logger styling

### What is the context of this PR?

The logs produced by Author and the API can be confusing to read when debugging as they aren't styled very well. I've introduced a prettier that'll only run locally with the hope that it'll make debugging easier.

I've excluded publisher from this as we're currently rebuilding it.

### How to review 

* Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
    * ensure it can be opened in Author;
    * then, ensure it can be viewed in Runner by pressing the **view survey** button
* Does this require a migration? We need one if existing JSON schema properties change
* Make the code looks OK
* Make sure the logs look OK

### What to do after everything is green

1. * [ ] Bring this branch up-to-date with Origin/Master
2. * [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. * [ ] Click the **Merge** button on this pull request
4. * [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. * [ ] Move the Trello card for this task into the next stage of the process
